apps/web/src/pages/EmployeeProfile/EmployeeProfilePage.tsx

```diff
@@ -212,7 +226,10 @@ const EmployeeProfilePage = () => {
   return (
     <Pending fetching={fetching} error={error}>
       {data?.me ? (
-        <EmployeeProfile userQuery={data.me} />
+        <EmployeeProfile
+          userQuery={data.me}
+          careerDevelopmentOptionsQuery={data}
```

Comment:
`data` already has me inside of it. I don't think you need it twice in here.  Remove `userQuery` from the props and you can it from `careerDevelopmentOptionsQuery` inside the `EmployeeProfile` component.

Reply:
Hmm, I'm not sure what you mean here? I kept them separate since they are both fragments that will be used in the nested component.

Reply:
This makes sense.  One is a fragment on `User` and one is a fragment on `Query`, right?